src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: () => number;
```

Comment:
The 'getArea' method in the 'Figure' interface should not accept any parameters. The method should use object fields to calculate the area.